src/main/java/tasker/tasker/service/UserService.java

```diff
@@ -86,18 +92,34 @@ public UserListDTO convertToListDto(User user) {
         return dto;
     }
 
-    public UserPageDTO convertToPageDto(Long id) {
+    public UserPageDto convertToPageDto(Long id) {
         User user = this.findUserById(id);
-        UserPageDTO dto = new UserPageDTO();
+        UserPageDto dto = new UserPageDto();
 
         dto.setId(user.getId());
         dto.setFirstName(user.getFirstName());
         dto.setLastName(user.getLastName());
         dto.setEmail(user.getEmail());
         dto.setUpdatedAt(user.getUpdatedAt());
-        dto.setActive(user.isActive());
+        dto.setActive(user.getActive());
         dto.setTeam(user.getTeam());
 
         return dto;
     }
+
+    public TaskListDto convertToTaskListDto(Task task) {
```

Comment:
consider using mapping libs

please refer to the following pages:
https://github.com/orika-mapper/orika
https://www.baeldung.com/rest-api-pagination-in-spring